typedb-src/modules/ROOT/partials/linux.adoc

```diff
@@ -69,6 +69,9 @@ specific version for every package.
 
 // tag::install-manual[]
 
+`ARM64` architecture support on Linux OS will be added in future versions of TypeDB.
```

Comment:
Hang on - is this for native? Our native distributions supports both Linux ARM and Mac ARM. Only docker, which is based on Linux, is x86-64 only.

Reply:
Oh, so we do support Linux `ARM64` native installation? I thought it was not the case. You were saying something about Cloud VMs on Linux available on ARMs. It was only for Docker, then?
```suggestion
```